pgml-dashboard/src/utils/url_parser.rs

```diff
@@ -1,30 +1,43 @@
 use lazy_static::lazy_static;
 use regex::Regex;
+use url::Url;
```

Comment:
I'd be curious to know why we need a url parser. I think Rocket should be able to provide you with everything you need to extract the uuid or anything else you need.